### PR TITLE
Added contraints V2

### PR DESCRIPTION
**Pull request by rubenglobant**

### CODE REVIEW

**Comment by rubenglobant** [comment]
added constraints according Juan Carlos suggestions

File: INActors/ActorsTableViewController.swift [hunk 1]
```diff
@@ -28,10 +28,15 @@ class ActorsTableViewController: UITableViewController, UISearchControllerDelega
     }
     
     // hide keyboard when searh ends
-    func searchBarSearchButtonClicked(_ searchBar: UISearchBar) {
-        self.actorsSearchBar.endEditing(true)
+//    func searchBarSearchButtonClicked(_ searchBar: UISearchBar) {
```
Comment (jsamboni) [inline INActors/ActorsTableViewController.swift]: if we're not using this, we should delete it

File: INActors/ActorsTableViewController.swift [hunk 2]
```diff
@@ -117,20 +122,34 @@ class ActorsTableViewController: UITableViewController, UISearchControllerDelega
...
+        description.getMoviePoster = moviePoster
         
         // send data to ActorsDescriptionViewController
         self.navigationController?.pushViewController(description, animated: true)
 
     }
 
 }
+
+
```
Comment (jsamboni) [inline INActors/ActorsTableViewController.swift]: extra line break should be deleted

**Review by jsamboni — CHANGES_REQUESTED** [review]
Some minor changes to be made, but the constraints looks good!

**Review by jsamboni — APPROVED** [review]
Looks nice!

**Comment by rubenglobant** [comment]
merge 1

**Pull request merged**